src/main/kotlin/no/skatteetaten/aurora/gorg/extensions/BuildConfig.kt

```diff
@@ -0,0 +1,17 @@
+package no.skatteetaten.aurora.gorg.extensions
+
+import io.fabric8.openshift.api.model.BuildConfig
+import no.skatteetaten.aurora.gorg.service.BuildConfigResource
+import java.time.Duration
+import java.time.Instant
+
+fun BuildConfig.toResource(now: Instant): BuildConfigResource {
```

Comment:
Perhaps we can merge some of the files in the `extensions` package?
Seems like many of them work on similar object, from kubernetes-model

Reply:
In other of our projects we put extention methods into a file that is named after the class that we extend. I feel it is easier to find them that way. I am open to joining them on functionality/group though.